omnibus_overrides.rb

```diff
@@ -9,8 +9,8 @@
 override "chef-workstation-app", version: "v0.1.77"
 # /DO NOT MODIFY
 
-override :rubygems, version: "3.0.3" # rubygems ships its own bundler which may differ from bundler defined below and then we get double bundler which results in performance issues / CLI warnings. Make sure these versions match before bumping either.
-override :bundler, version: "1.17.2" # currently pinned to what ships in Ruby to prevent double bundler
+override :rubygems, version: "3.1.2" # rubygems ships its own bundler which may differ from bundler defined below and then we get double bundler which results in performance issues / CLI warnings. Make sure these versions match before bumping either.
+override :bundler, version: "2.1.4" # currently pinned to what ships in Ruby to prevent double bundler
```

Comment:
Hmmm, this one we may need to discuss. I know there was an issue with updating to > 2.x and I'm not sure if that issue still exists.

Reply:
This is what Chef is currently doing so we do that too